usabilla-react-native/README.md

```diff
@@ -40,13 +43,39 @@ react-native link usabilla-react-native
 To use this Bridge, please make sure you are using XCode 9.1 or above.
 
 ## Features
-The first version of Usabilla for React Native `0.1.0` allows you to :
-- Load and show a Passive Feedback form.
-- Submit the results of the form.
+### Load and show a Passive Feedback form
+In order to load a passive feedback form with the  Usabilla library few methods need to be implemented:
+- `usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")`
+  - The method performing the call for the form
+- `usabilla.setFormDidLoadSuccessfully(usabilla.showLoadedForm())`
+  - The method setting the callback to execute when the form is fetched successfully
+- `usabilla.setFormDidFailLoading((error) => console.log("Error loading form:", error))`
+  - The method setting the callback to execute when the form fails to be fetched. This callback has a parameter `error` contaning a textual explanation of what happened
+- **[iOS ONLY]** `usabilla.setFormDidClose((reminder) => console.log(reminder.formId))`
+  - The method setting the callback to execute when the form closes due to completion or cancellation by the user. This callback has a parameter `reminder` contaning:
+    - formId (string)
+    - feedbackResults ([FeedbackResults](https://github.com/usabilla/usabilla-u4a-android-sdk#feedback-result))
+    - isRedirectToAppStoreEnabled (boolean)
+
+### Submit the results of the form
+This functionality is embedded in the native Usabilla library and there is no need to perform any specific action from the React Native environment.
+
+### Support for custom variables
+
+In order to set custom variables in the Usabilla native library it's necessary to call the method:
+- `usabilla.setCustomVariables(customVars)`
+with the parameter `customVars` being a valid JSON containing only values of type:
+  - String
+  - Number
+  - Boolean
+  - Array (of single type either Strings, Numbers or Booleans)
+  
+  i.e. `var customVars = {"flight-number": "99999999", "user-type": "premium", "number-flights": 12}`
+
+**WARNING**: Objects, arrays of Objects and types not listed above are not supported
```

Comment:
This is not needed! If it's a json object then we get this by default

Reply:
Yes, a json object can contain other objects